### ISSUE TITLE
App Manager displays only seven apps instead of all of them

### ISSUE THREAD

**Issue by 2011** [MuntashirAkon/AppManager#167]
**Describe the bug**

First, App Manager shows tremendous potential, and I want to thank you for your efforts.

I see this problem on only one (so far) of three (so far) devices.

The first time I started App Manager, it appeared to work normally (displaying all of the apps on the device). I changed a few of the settings (Night theme, Root mode). The second (and subsequent) time(s) I started App Manager, it displayed only seven apps (no I do not have any filters checked). The apps displayed seem to have no pattern. I don't want to mention them all here for privacy reasons, but they include Calendar Storage, Mobile Network Configuration, and OpenVPN for Android. The others consist of a shopping app, a torrent app, a media app, and a messaging app. I have tried the obvious possible solution of clearing data and cache, but that had no effect. I still see just a tiny percentage of the apps.

**To Reproduce**

No idea. I only observe this on one machine, and doubt I could make this happen anywhere else.

**Expected behavior**

I expect to see all of the apps, not just a handful. :)

**Screenshots**

I don't want to post these publicly (again for privacy reasons), but could send them directly if you believe they would help find the source of the problem (I don't know that they would - a screenshot would just show a short list of apps).

**Crash logs**

The program did crash a couple of times before starting the second time. I don't know if that has any connection with this problem or not. I didn't save any logs then (not really sure how to do that for this program), but I could attempt to obtain them if given some guidance.

**Device info**

 - Device: Samsung Galaxy Tab 2
 - OS Version: LineageOS 13.0-20180211-NIGHTLY-espressowifi
 - App Manager Version: 2.5.17 (368)
 - Mode: root

**Comment by 2011** [comment]
Can you tell me exactly which information you would find most helpful?

**Comment by 2011** [comment]
A little more information:

If I select "Running Apps" in the menu, I do see all of the running apps.

Also, the app crashes (100 perecent of the time) by selecting "1-Click Ops," then "Block/unblock trackers," and then "no" (the app crashes during the search shortly after selecting "no."

The relevant portion of the logcat:

11-28 13:37:35.910  5130  5408 E AndroidRuntime: FATAL EXCEPTION: Thread-356
11-28 13:37:35.910  5130  5408 E AndroidRuntime: Process: io.github.muntashirakon.AppManager, PID: 5130
11-28 13:37:35.910  5130  5408 E AndroidRuntime: Theme: themes:{}
11-28 13:37:35.910  5130  5408 E AndroidRuntime: java.lang.RuntimeException: Package manager has died
11-28 13:37:35.910  5130  5408 E AndroidRuntime:        at android.app.ApplicationPackageManager.getPackageInfo(ApplicationPackageManager.java:135)
11-28 13:37:35.910  5130  5408 E AndroidRuntime:        at io.github.muntashirakon.AppManager.utils.PackageUtils.collectComponentClassNames(PackageUtils.java:98)
11-28 13:37:35.910  5130  5408 E AndroidRuntime:        at io.github.muntashirakon.AppManager.rules.compontents.ComponentUtils.getTrackerComponentsForPackage(ComponentUtils.java:65)
11-28 13:37:35.910  5130  5408 E AndroidRuntime:        at io.github.muntashirakon.AppManager.rules.compontents.ComponentUtils.getTrackerCountForApp(ComponentUtils.java:147)
11-28 13:37:35.910  5130  5408 E AndroidRuntime:        at io.github.muntashirakon.AppManager.oneclickops.OneClickOpsActivity.lambda$blockTrackers$11$OneClickOpsActivity(OneClickOpsActivity.java:122)
11-28 13:37:35.910  5130  5408 E AndroidRuntime:        at io.github.muntashirakon.AppManager.oneclickops.-$$Lambda$OneClickOpsActivity$vtrzWGnwdrjybe0NvkVXmfS-A5U.run(lambda)
11-28 13:37:35.910  5130  5408 E AndroidRuntime:        at java.lang.Thread.run(Thread.java:818)
11-28 13:37:35.910  5130  5408 E AndroidRuntime: Caused by: android.os.DeadObjectException: Transaction failed on small parcel; remote process probably died
11-28 13:37:35.910  5130  5408 E AndroidRuntime:        at android.os.BinderProxy.transactNative(Native Method)
11-28 13:37:35.910  5130  5408 E AndroidRuntime:        at android.os.BinderProxy.transact(Binder.java:503)
11-28 13:37:35.910  5130  5408 E AndroidRuntime:        at android.content.pm.IPackageManager$Stub$Proxy.getPackageInfo(IPackageManager.java:2247)
11-28 13:37:35.910  5130  5408 E AndroidRuntime:        at android.app.ApplicationPackageManager.getPackageInfo(ApplicationPackageManager.java:130)
11-28 13:37:35.910  5130  5408 E AndroidRuntime:        ... 6 more

**Comment by 2011** [comment]
Another crash when working with one of the diplayed app.

11-29 02:37:10.557  1330  1330 E AndroidRuntime: FATAL EXCEPTION: main
11-29 02:37:10.557  1330  1330 E AndroidRuntime: Process: io.github.muntashirakon.AppManager, PID: 1330
11-29 02:37:10.557  1330  1330 E AndroidRuntime: Theme: themes:{}
11-29 02:37:10.557  1330  1330 E AndroidRuntime: java.lang.RuntimeException: Error receiving broadcast Intent { act=android.intent.action.PACKAGE_CHANGED dat=package:[removed for privacy reasons] flg=0x4000010 (has extras) } in io.github.muntashirakon.AppManager.main.MainViewModel$PackageIntentReceiver@321b2f8
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        at android.app.LoadedApk$ReceiverDispatcher$Args.run(LoadedApk.java:892)
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        at android.os.Handler.handleCallback(Handler.java:739)
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:95)
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:148)
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:5461)
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
11-29 02:37:10.557  1330  1330 E AndroidRuntime: Caused by: java.lang.RuntimeException: Package manager has died
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        at android.app.ApplicationPackageManager.getPackageInfo(ApplicationPackageManager.java:135)
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        at io.github.muntashirakon.AppManager.main.MainViewModel.getNewApplicationItem(MainViewModel.java:425)
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        at io.github.muntashirakon.AppManager.main.MainViewModel.updateInfoForPackages(MainViewModel.java:376)
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        at io.github.muntashirakon.AppManager.main.MainViewModel.updateInfoForUid(MainViewModel.java:356)
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        at io.github.muntashirakon.AppManager.main.MainViewModel.access$000(MainViewModel.java:61)
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        at io.github.muntashirakon.AppManager.main.MainViewModel$PackageIntentReceiver.onReceive(MainViewModel.java:510)
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        at android.app.LoadedApk$ReceiverDispatcher$Args.run(LoadedApk.java:882)
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        ... 7 more
11-29 02:37:10.557  1330  1330 E AndroidRuntime: Caused by: android.os.DeadObjectException: Transaction failed on small parcel; remote process probably died
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        at android.os.BinderProxy.transactNative(Native Method)
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        at android.os.BinderProxy.transact(Binder.java:503)
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        at android.content.pm.IPackageManager$Stub$Proxy.getPackageInfo(IPackageManager.java:2247)
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        at android.app.ApplicationPackageManager.getPackageInfo(ApplicationPackageManager.java:130)
11-29 02:37:10.557  1330  1330 E AndroidRuntime:        ... 13 more

**Comment by MuntashirAkon** [comment]
DeadObjectException is thrown by the system when the app has reached the limit to call `PackageManager#getPackageInfo` or `PackageManager#getApplicationInfo` for a certain period. The affected Android versions are Android 5-7, I believe.

**Comment by MuntashirAkon** [comment]
And please download and install the latest debug builds (they have different package names) for reporting crashes.

**Comment by 2011** [comment]
> DeadObjectException is thrown by the system when the app has reached the limit to call PackageManager#getPackageInfo or PackageManager#getApplicationInfo for a certain period. The affected Android versions are Android 5-7, I believe.

Yes, this LineageOS version comes from Android 6.0.1 (should have mentioned that in the initial report).

> And please download and install the latest debug builds (they have different package names) for reporting crashes.

Where do I find the debug builds? I see a 2.5.18 "Pre-release." Do you mean that?

**Comment by MuntashirAkon** [comment]
> Where do I find the debug builds? I see a 2.5.18 "Pre-release." Do you mean that?

No. You'll find debug builds in the GitHub Actions: https://github.com/MuntashirAkon/AppManager/actions

**Comment by 2011** [comment]
> > Where do I find the debug builds? I see a 2.5.18 "Pre-release." Do you mean that?
> 
> No. You'll find debug builds in the GitHub Actions: https://github.com/MuntashirAkon/AppManager/actions

Thanks for that (didn't know anything about those). The good news - neither of the two actions that caused crashes in the release version do so in the debug version. The bad news - the main display still shows only the same seven apps as before.

**Comment by MuntashirAkon** [comment]
Please try the latest debug build again.

**Comment by 2011** [comment]
Still (from Debug Build 487) the same seven apps displayed (but no crashes). Does some way exist to get additional information from the Debug Build?

**Comment by MuntashirAkon** [comment]
You don't experience crashes because they are being suppressed. This is an inherent limitation to PackageManager. All I can do is to prevent restarting the app when app local has changed and 19217175 should prevent restarting activities when app locale has changed which, unfortunately, isn't working for you.

> Does some way exist to get additional information from the Debug Build?

Yeah. There exists a log file in the following path but it only logs the last launch.
```
/sdcard/Android/data/io.github.muntashirakon.AppManager.debug/files/cache/am.log
```

**Comment by 2011** [comment]
Not much in there, but in case it helps:

2020-11-30 07:12:14.668 D/MVM: New instance created
2020-11-30 07:12:14.846 D/Runner: RootShellRunner
2020-11-30 07:12:15.194 D/backupApplications: []
2020-11-30 07:12:15.898 D/MVM: New instance created
2020-11-30 07:12:16.051 D/backupApplications: []

**Comment by MuntashirAkon** [comment]
Yes, I didn't think they would help. Thanks anyway.

**Comment by MuntashirAkon** [comment]
@2011 Can you still reproduce it in v2.5.21?

**Comment by MuntashirAkon** [comment]
Closed due to inactivity. Feel free to open it again if the issue still exists.

**Comment by 2011** [comment]
@MuntashirAkon F-Droid hasn't updated this in several months (I sort of kept waiting for updates from them), but I just downloaded the newly released 2.6.0 from this site, and it has the same problem (not displaying all apps) as before.

**Comment by MuntashirAkon** [comment]
> F-Droid hasn't updated this in several months (I sort of kept waiting for updates from them), but I just downloaded the newly released 2.6.0 from this site, and it has the same problem (not displaying all apps) as before.

Take logs from logcat when you open the app.

**Comment by 2011** [comment]
Lots of stuff there, but here goes:

```
[05-01 11:20:47.283 468:696 I/ActivityManager]
START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=io.github.muntashirakon.AppManager/.main.MainActivity bnds=[300,30][400,186] (has extras)} from uid 10028 on display 0

[05-01 11:20:47.332 468:482 W/BroadcastQueue]
Permission Denial: receiving Intent { act=com.android.launcher3.action.LAUNCH flg=0x10 (has extras) } to com.google.android.gms/.chimera.GmsIntentOperationService$GmsExternalReceiver requires com.android.launcher3.permission.RECEIVE_LAUNCH_BROADCASTS due to sender com.cyanogenmod.trebuchet (uid 10028)

[05-01 11:20:47.366 468:1950 I/ActivityManager]
Start proc 7432:io.github.muntashirakon.AppManager/u0a107 for activity io.github.muntashirakon.AppManager/.main.MainActivity

[05-01 11:20:47.921 7432:7432 E/BypassProvider]
Unable to unseal hidden api access
java.lang.reflect.InvocationTargetException
    at java.lang.reflect.Method.invoke(Native Method)
    at org.chickenhook.restrictionbypass.RestrictionBypass.getDeclaredMethod(RestrictionBypass.java:18)
    at org.chickenhook.restrictionbypass.Unseal.unseal(Unseal.java:15)
    at org.chickenhook.restrictionbypass.BypassProvider.attachInfo(BypassProvider.java:51)
    at android.app.ActivityThread.installProvider(ActivityThread.java:5197)
    at android.app.ActivityThread.installContentProviders(ActivityThread.java:4792)
    at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4732)
    at android.app.ActivityThread.-wrap1(ActivityThread.java)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1424)
    at android.os.Handler.dispatchMessage(Handler.java:102)
    at android.os.Looper.loop(Looper.java:148)
    at android.app.ActivityThread.main(ActivityThread.java:5461)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
Caused by: java.lang.NoSuchMethodException: setHiddenApiExemptions [class [Ljava.lang.String;]
    at java.lang.Class.getMethod(Class.java:624)
    at java.lang.Class.getDeclaredMethod(Class.java:586)
    ... 15 more

[05-01 11:20:48.575 7432:7432 W/art]
Before Android 4.1, method android.graphics.PorterDuffColorFilter androidx.vectordrawable.graphics.drawable.VectorDrawableCompat.updateTintFilter(android.graphics.PorterDuffColorFilter, android.content.res.ColorStateList, android.graphics.PorterDuff$Mode) would have incorrectly overridden the package-private method in android.graphics.drawable.Drawable

[05-01 11:20:48.673 468:7286 I/ActivityManager]
START u0 {cmp=io.github.muntashirakon.AppManager/.crypto.AuthenticationActivity} from uid 10107 on display 0

[05-01 11:20:48.796 7432:7452 D/OpenGLRenderer]
Use EGL_SWAP_BEHAVIOR_PRESERVED: true

[05-01 11:20:48.804 7432:7432 W/linker]
/system/vendor/lib/hw/gralloc.omap4.so: unused DT entry: type 0xf arg 0x53b

[05-01 11:20:48.808 7432:7432 W/linker]
/system/vendor/lib/libpvr2d.so: unused DT entry: type 0xf arg 0x767

[05-01 11:20:48.957 7432:7452 I/OpenGLRenderer]
Initialized EGL, version 1.4

[05-01 11:20:48.958 7432:7452 W/OpenGLRenderer]
Failed to choose config with EGL_SWAP_BEHAVIOR_PRESERVED, retrying without...

[05-01 11:20:49.041 468:7285 I/ActivityManager]
START u0 {cmp=io.github.muntashirakon.AppManager/.crypto.AuthenticationActivity} from uid 10107 on display 0

[05-01 11:20:49.551 468:482 W/ActivityManager]
Activity pause timeout for ActivityRecord{278f01e u0 io.github.muntashirakon.AppManager/.crypto.AuthenticationActivity t231}

[05-01 11:20:49.651 7432:7456 E/DataTransmission]
shakeHands: Token: d7868c764e86d3f7360bfde11af8e990, Client protocol: 1.2.4

[05-01 11:20:49.678 7432:7443 I/art]
Background partial concurrent mark sweep GC freed 9847(1056KB) AllocSpace objects, 6(120KB) LOS objects, 40% free, 3MB/6MB, paused 11.016ms total 160.308ms

[05-01 11:20:50.176 141:175 E/BufferQueueProducer]
[io.github.muntashirakon.AppManager/io.github.muntashirakon.AppManager.crypto.AuthenticationActivity] allocateBuffers: slot 2 without buffer is not FREE

[05-01 11:20:51.023 141:141 W/SurfaceFlinger]
couldn't log to binary event log: overflow.

[05-01 11:20:52.170 7468:7468 D/AndroidRuntime]
>>>>>> START com.android.internal.os.RuntimeInit uid 0 <<<<<<

[05-01 11:20:52.211 7468:7468 D/AndroidRuntime]
CheckJNI is OFF

[The log then contains about 600 "W/art" entries, of which I will include just the first few and last few, as they don't appear particularly relevant. If you need them, I can add them later]

[05-01 11:20:52.277 7468:7468 W/art]
70bcd000-715cf000 rw-p 00000000 103:02 497764    /data/dalvik-cache/arm/system@framework@boot.art

[05-01 11:20:52.277 7468:7468 W/art]
b10a5000-b3027000 r--p 00000000 103:01 65289     /system/framework/arm/boot.oat

[05-01 11:20:52.277 7468:7468 W/art]
b3027000-b47ed000 r-xp 01f82000 103:01 65289     /system/framework/arm/boot.oat

[05-01 11:20:52.277 7468:7468 W/art]
b47ed000-b47ee000 rw-p 03748000 103:01 65289     /system/framework/arm/boot.oat

[05-01 11:20:52.277 7468:7468 W/art]
b47ee000-b4c33000 r-xp 00000000 103:01 73499     /system/lib/libart.so

[...]

[05-01 11:20:52.683 7477:7477 W/art]
32

[05-01 11:20:52.683 7477:7477 W/art]
b6fa1000-b6fa2000 rw-p 00000000 00:00 0 

[05-01 11:20:52.683 7477:7477 W/art]
beb3f000-beb60000 rw-p 00000000 00:00 0          [stack]

[05-01 11:20:52.683 7477:7477 W/art]
ffff0000-ffff1000 r-xp 00000000 00:00 0          [vectors]

[05-01 11:20:52.683 7477:7477 W/art]


[05-01 11:20:52.806 7468:7468 D/ICU]
No timezone override file found: /data/misc/zoneinfo/current/icu/icu_tzdata.dat

[05-01 11:20:52.891 7477:7477 D/ICU]
No timezone override file found: /data/misc/zoneinfo/current/icu/icu_tzdata.dat

[05-01 11:20:53.114 7477:7477 E/memtrack]
Couldn't load memtrack module (No such file or directory)

[05-01 11:20:53.114 7477:7477 E/android.os.Debug]
failed to load memtrack module: -2

[05-01 11:20:53.120 7468:7468 E/memtrack]
Couldn't load memtrack module (No such file or directory)

[05-01 11:20:53.120 7468:7468 E/android.os.Debug]
failed to load memtrack module: -2

[05-01 11:20:53.161 7468:7468 I/Radio-JNI]
register_android_hardware_Radio DONE

[05-01 11:20:53.161 7477:7477 I/Radio-JNI]
register_android_hardware_Radio DONE

[05-01 11:20:53.224 7468:7468 D/AndroidRuntime]
Calling main entry com.android.commands.content.Content

[05-01 11:20:53.230 7477:7477 D/AndroidRuntime]
Calling main entry io.github.muntashirakon.AppManager.server.IPCMain

[05-01 11:20:53.385 7492:7492 D/idmap]
error: no read access to /vendor/overlay: No such file or directory

[05-01 11:20:53.629 7477:7477 I/art]
System.exit called, status: 0

[05-01 11:20:53.629 7477:7477 I/AndroidRuntime]
VM exiting with result code 0.

[05-01 11:20:53.640 3378:3378 D/IPC]
ComponentInfo{io.github.muntashirakon.AppManager/io.github.muntashirakon.AppManager.ipc.AMService} rebind

[05-01 11:20:53.641 3378:3378 D/IPC]
AMService: onBind

[05-01 11:20:53.710 7491:7504 I/art]
Background sticky concurrent mark sweep GC freed 9449(470KB) AllocSpace objects, 0(0B) LOS objects, 57% free, 2MB/4MB, paused 3.631m
s total 119.934ms

[05-01 11:20:53.851 7432:7432 W/art]
Before Android 4.1, method android.graphics.drawable.Drawable com.google.android.material.progressindicator.BaseProgressIndicator.ge
tCurrentDrawable() would have incorrectly overridden the package-private method in android.widget.ProgressBar

[05-01 11:20:54.357 7468:7468 D/AndroidRuntime]
Shutting down VM

[05-01 11:20:54.405 3378:3390 D/IPC]
transact: uid=10107, code=6

[05-01 11:20:54.408 3378:3412 D/IPC]
transact: uid=10107, code=6

[05-01 11:20:54.410 3378:3391 D/IPC]
transact: uid=10107, code=6

[D/IPC messages repeat 13 times in total]

[...]

[05-01 11:20:54.448 7491:7528 D/OpenGLRenderer]
Use EGL_SWAP_BEHAVIOR_PRESERVED: true

[05-01 11:20:54.461 7491:7491 W/linker]
/system/vendor/lib/hw/gralloc.omap4.so: unused DT entry: type 0xf arg 0x53b

[05-01 11:20:54.463 7491:7491 W/linker]
/system/vendor/lib/libpvr2d.so: unused DT entry: type 0xf arg 0x767

[05-01 11:20:54.587 3378:3412 D/IPC]
transact: uid=10107, descriptor=android.os.IUserManager, code=9

[05-01 11:20:54.609 3378:3390 D/IPC]
transact: uid=10107, descriptor=android.content.pm.IPackageManager, code=49

[05-01 11:20:54.616 7491:7528 I/OpenGLRenderer]
Initialized EGL, version 1.4

[05-01 11:20:54.617 7491:7528 W/OpenGLRenderer]
Failed to choose config with EGL_SWAP_BEHAVIOR_PRESERVED, retrying without...

[05-01 11:20:54.651 7432:7432 I/Choreographer]
Skipped 55 frames!  The application may be doing too much work on its main thread.

[05-01 11:20:54.712 7491:7528 V/RenderScript]
0xa04a5000 Launching thread(s), CPUs 2

[05-01 11:20:54.824 141:141 W/SurfaceFlinger]
couldn't log to binary event log: overflow.

[05-01 11:20:54.942 7432:7443 I/art]
Background sticky concurrent mark sweep GC freed 12183(866KB) AllocSpace objects, 6(152KB) LOS objects, 13% free, 5MB/6MB, paused 1.220ms total 107.025ms

[05-01 11:20:54.945 468:478 I/art]
Background sticky concurrent mark sweep GC freed 69390(3MB) AllocSpace objects, 3(60KB) LOS objects, 10% free, 19MB/22MB, paused 9.521ms total 245.849ms

[05-01 11:20:55.048 7432:7530 E/JavaBinder]
!!! FAILED BINDER TRANSACTION !!!  (parcel size = 4)

[05-01 11:20:55.077 7432:7530 W/ParceledListSlice]
Failure retrieving array; only received 5 of 181
android.os.DeadObjectException: Transaction failed on small parcel; remote process probably died
    at android.os.BinderProxy.transactNative(Native Method)
    at android.os.BinderProxy.transact(Binder.java:503)
    at android.content.pm.ParceledListSlice.<init>(ParceledListSlice.java:97)
    at android.content.pm.ParceledListSlice.<init>(ParceledListSlice.java)
    at android.content.pm.ParceledListSlice$1.createFromParcel(ParceledListSlice.java:200)
    at android.content.pm.ParceledListSlice$1.createFromParcel(ParceledListSlice.java:199)
    at android.content.pm.IPackageManager$Stub$Proxy.getInstalledPackages(IPackageManager.java:3279)
    at io.github.muntashirakon.AppManager.servermanager.PackageManagerCompat.getInstalledPackages(PackageManagerCompat.java:71)
    at io.github.muntashirakon.AppManager.utils.PackageUtils.updateInstalledOrBackedUpApplications(PackageUtils.java:258)
    at io.github.muntashirakon.AppManager.utils.PackageUtils.lambda$getInstalledOrBackedUpApplicationsFromDb$0(PackageUtils.java:170)
    at io.github.muntashirakon.AppManager.utils.-$$Lambda$PackageUtils$FDXMpcfHZMTjLENDE7nHv1buIR0.run(lambda)
    at java.lang.Thread.run(Thread.java:818)

[The following pattern repeats indefinitely]

[05-01 11:20:55.275 3378:3390 D/IPC]
transact: uid=10107, code=6

[05-01 11:20:55.279 3378:3412 D/IPC]
transact: uid=10107, code=6

[05-01 11:20:55.281 3378:3391 D/IPC]
transact: uid=10107, code=6

[05-01 11:20:55.282 3378:4941 D/IPC]
transact: uid=10107, code=6

[05-01 11:20:55.282 3378:3390 D/IPC]
transact: uid=10107, code=6

[05-01 11:20:55.283 3378:4941 D/IPC]
transact: uid=10107, descriptor=android.content.pm.IPackageManager, code=3
```

**Comment by MuntashirAkon** [comment]
Duplicate of #298
